tutorials/iot/ubuntu-kiosk/wayland-kiosk.md

```diff
@@ -504,34 +475,68 @@ root@in-snap:~# $SNAP/command-glmark2-example.wrapper
 Oh, still fails with a bunch of errors like
 
 
+```
+Error: eglGetDisplay() failed with error: 0x3000
+Error: eglGetDisplay() failed with error: 0x3000
+Error: main: Could not initialize canvas
+```
+
+
+Courage! This is progress, we’re almost done in fact. There’s just one more thing to fix…
+
+positive
+: Note for when using “core” (aka “core16”) as the base
```

Comment:
It is tempting to offer options, but it is not appropriate for an initial tutorial: options are a source of confusion. We should present "steps that will work" with as few distractions as possible. (I once had a bad dance teacher that used to say "you can do this... or this... or this..." and, as a result, nobody knew what to do.)

I'd drop everything to do with core16 from the tutorials.

Reply:
I definitely get your point, I was torn about this, so thought best compromise was to put any core16-specific details in but as small as possible.

But I thought it was worth including due to (a) there are several non-trivial differences and (b) I expect core16 isn't going away any time soon.

However I will remove if you'd prefer.

Reply:
We don't need core16: the Wayland support in bionic toolkits is problematic.

> I'd drop everything to do with core16 from the tutorials.

Reply:
All removed